compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -2771,3 +2774,54 @@ fn ctstring_eq(arguments: Vec<(Value, Location)>, location: Location) -> IResult
 fn ctstring_hash(arguments: Vec<(Value, Location)>, location: Location) -> IResult<Value> {
     hash_item(arguments, location, get_ctstring)
 }
+
+fn derive_generators(
+    interner: &mut NodeInterner,
+    arguments: Vec<(Value, Location)>,
+    return_type: Type,
+    location: Location,
+) -> IResult<Value> {
+    assert_eq!(arguments.len(), 2);
```

Comment:
```suggestion
```
This isn't guaranteed and `check_two_arguments` should already error if there aren't two arguments